Watchman.Discord/Areas/Users/Controllers/UsersController.cs

```diff
@@ -44,10 +49,24 @@ public void GetAvatar(DiscordRequest request, Contexts contexts)
         [DiscordCommand("add role")] //todo
         public void AddRole(DiscordRequest request, Contexts contexts)
         {
-            var commandRole = request.OriginalMessage.ToLowerInvariant().Replace("-add role ", string.Empty); //TODO use DiscordRequest properties
-            var safeRoles = this._queryBus.Execute(new GetDiscordServerSafeRolesQuery(contexts.Server.Id)).SafeRoles;
-            var messagesService = _messagesServiceFactory.Create(contexts);
-            _rolesService.AddRoleToUser(safeRoles, messagesService, contexts, commandRole);
+            var requestArguments = request.Arguments.Skip(1);
+            var rolesToAssign = requestArguments.Select(x => x.Value);
+
```

Comment:
ta pusta linia nie ma tu za bardzo sensu, bo po niej masz operacje na zmiennej którą właśnie stworzyłeś
to nie jest kawałek kodu który powinien być jakoś wyraźnie odznaczony